constantine/serialization/codecs_banderwagon.nim

```diff
@@ -110,4 +110,44 @@ func deserialize*(dst: var EC_Prj, src: array[32, byte]): CttCodecEccStatus =
   if not(bool dst.isInSubgroup()):
     return cttCodecEcc_PointNotInSubgroup
 
-  return cttCodecEcc_Success
\ No newline at end of file
+  return cttCodecEcc_Success
+
+## ############################################################
+##
+##              Banderwagon Batch Serialization
+##
+## ############################################################
+
+func serializeBatch*(
+    dst: ptr UncheckedArray[array[32, byte]],
+    points: ptr UncheckedArray[EC_Prj],
+    N: static int,
```

Comment:
It's very likely that N won't be known at compile-time and should be a regular int.

Reply:
And then we can add a wrapper for arrays.